src/field_theory/separable.lean

```diff
@@ -68,6 +69,24 @@ end
 lemma separable.of_mul_right {f g : polynomial R} (h : (f * g).separable) : g.separable :=
 by { rw mul_comm at h, exact h.of_mul_left }
 
+lemma separable_gcd_left {F : Type*} [field F] (f g : polynomial F) (hf : f.separable) :
+  (euclidean_domain.gcd f g).separable :=
+begin
+  cases euclidean_domain.gcd_dvd_left f g with p mul,
+  apply polynomial.separable.of_mul_left,
+  rw ←mul,
+  exact hf,
+end
+
+lemma separable_gcd_right {F : Type*} [field F] (f g : polynomial F) (hf : g.separable) :
```

Comment:
* `f` respectively `g` can be implicit parameters in these lemmas

* If you move these after `separable.of_div`, the proof can be simplified to `exact separable.of_div hf (euclidean_domain.gcd_dvd_left f g)`.